package.json

```diff
@@ -30,10 +30,10 @@
   "resolutions": {
     "ast-types": "^0.14.0",
     "node-sass": "^6.0.0",
-    "react": "~16.9.0",
-    "react-dom": "~16.9.0",
-    "react-is": "~16.9.0",
-    "react-test-renderer": "~16.9.0"
+    "react": "~17.0.2",
+    "react-dom": "~17.0.2",
+    "react-is": "~17.0.2",
+    "react-test-renderer": "~17.0.2"
```

Comment:
@joshblack is there any problem you know of if we update the `resolutions` here? I don't think it should have any impact on downstream consumers, but let me know if I should revert this.

Reply:
Should be good! This will force one version for us in the monorepo so that we don't have to worry about multiple versions of React in the monorepo and the weird hook shenanigans that come from that.